codingTest_김준우/문자열 다루기 기본.js

```diff
@@ -0,0 +1,17 @@
+function solution(s) {
+  if (s.length === 4 || s.length === 6) {
+    let splitS = s.split("");
+    let answer = true;
+    for (let i = 0; i < splitS.length; i++) {
+      if (isNaN(splitS[i]) === true) {
+        answer = false;
+        return answer;
```

Comment:
```suggestion
      if (isNaN(splitS[i])   {
        return false
```
isNaN( value ) 메소드는 value값이 NaN이면 true를, 아니면 false를 반환하는 메소드기 때문에 굳이
isNaN이 true인지 false인지 조건을 주지 않아도 돼요